lib/cib/Makefile.am

```diff
@@ -12,8 +12,13 @@ include $(top_srcdir)/mk/common.mk
 lib_LTLIBRARIES		= libcib.la
 
 ## SOURCES
-libcib_la_SOURCES	= cib_ops.c cib_utils.c cib_client.c cib_native.c cib_attrs.c
-libcib_la_SOURCES	+= cib_file.c cib_remote.c
+libcib_la_SOURCES	= cib_ops.c 	\
+			  cib_utils.c 	\
+			  cib_client.c 	\
+			  cib_native.c 	\
+			  cib_attrs.c
+libcib_la_SOURCES	+= cib_file.c 	\
+			   cib_remote.c
```

Comment:
No need for the +=, it was just a way to break it up on two lines

Reply:
Fixed.

Reply:
?